src/kava.js

```diff
@@ -206,7 +207,7 @@ class Kava extends BasePlugin {
    */
   sendAnalytics(model: Object): Promise<*> {
     return new Promise((resolve, reject) => {
-      OVPAnalyticsService.trackEvent(this.config.serviceUrl, model)
+      OVPAnalyticsService.trackEvent(this.config.serviceUrl, model, this.config.requestMethod)
```

Comment:
@yairans does the service is case sensitive?

Reply:
yes

Reply:
I would suggest to add `toUpper...` and not to force it. 
Seems reasonable to pass also `get`/`post`.

Reply:
use an enum and expose it